src/tscov/tscov.ts

```diff
@@ -22,7 +22,12 @@ export class Tscov {
   ) {
     try {
       this.executeTscov()
-        .then(() => this.showSpacesLog())
+        .then(success => {
+          this.showSpacesLog()
+          if (!success && !this.suppressError) {
+            process.exit(1)
```

Comment:
I wasn't sure if the process exit should happen here or be hoisted up one caller. It's a little odd that a constructor would terminate a program. Though, it appears to be how it's designed when other errors happen.

See L34 of this file.

```js
if (!this.suppressError) {
  process.exit(1)
}
```

Reply:
Thanks for contributing!